www/docs/en/dev/config_ref/index.md

```diff
@@ -199,6 +199,18 @@ platform. See [Customize icons topic](images.html) for more information.
    Examples:
 
     <engine name="android" spec="^4.0.0" />
+    
+## plugin
+   Specifies what plugin to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html#plugin-versioning).
+
+   Attributes(type) | Description
+   ----------------- | ------------
+   name(string) | *Required* <br/> Name of the plugin to be restored
+   spec(string) | *Required* <br/> Version of the plugin to be restored
```

Comment:
Spec is not the `version` - correct? I describe it in detail here: https://cordova.apache.org/docs/en/dev/cordova-cli/index.html#plugin-spec